stdlib/float.mli

```diff
@@ -57,6 +57,13 @@ external mul : float -> float -> float = "%mulfloat"
 external div : float -> float -> float = "%divfloat"
 (** Floating-point division. *)
 
+external fma : float -> float -> float -> float =
+  "caml_fma_float" "caml_fma" [@@unboxed] [@@noalloc]
+(** [fma x y z] returns [x * y + z], with a best effort for computing
+    this expression with a single rounding, using either hardware
+    instructions (providing full IEEE compliance) or a software emulation.
+    @since 4.09.0 *)
```

Comment:
1. 4.08
2. I did not think about it before, but what about saying a word about performance (with and without emulation) compared to the naive version?  People might think that using fma is a good idea for performance reason even when precision is not a big concern, and it's not obvious whether this is a good idea.  Is it true that with hardware support, fma would be fastest but with emulation from libc or yours, itwould be slower?

Reply:
Add a note about performance.